tests/129_indexmap.rs

```diff
@@ -16,7 +16,7 @@ tasks: {
         "-h",
       ]),
       ch_dir: Some("/")
-    ),
```

Comment:
please change this back to emit the comma in case the elements are not all on the same line